lib/streamlit/hashing.py

```diff
@@ -413,7 +413,7 @@ def _code_to_bytes(self, code, context):
         else:
             # This won't correctly follow nested calls like `foo.bar.baz()`.
             for name in code.co_names:
-                if name in context.globals:
+                if name in context.globals and type(context.globals[name]) == "function":
```

Comment:
I thought you reverted this change?

Reply:
Good point. I'm confused too.

Reply:
I am also confused, because I thought I did...